eng/versioning/external_dependencies.txt

```diff
@@ -56,8 +56,8 @@ io.netty:netty-tcnative-boringssl-static;2.0.34.Final
 io.netty:netty-transport-native-epoll;4.1.53.Final
 io.netty:netty-transport-native-unix-common;4.1.53.Final
 io.netty:netty-transport-native-kqueue;4.1.53.Final
-io.projectreactor.netty:reactor-netty;0.9.13.RELEASE
-io.projectreactor:reactor-core;3.3.11.RELEASE
+io.projectreactor.netty:reactor-netty;0.9.15.RELEASE
```

Comment:
No updates on netty versions ?
This release uses 54.Final versions of netty. 
https://mvnrepository.com/artifact/io.projectreactor.netty/reactor-netty/0.9.15.RELEASE

Reply:
Good catch

Reply:
LGTM